webapp/docs/01_introduction_and_goals.adoc

```diff
@@ -30,6 +38,31 @@ If requirements documents exist this overview should refer to these documents.
 Keep these excerpts as short as possible. Balance readability of this document with potential redundancy w.r.t to requirements documents.
 ****
 
+https://arquisoft.github.io/course2021/labAssignmentDescription.html[The requirements for this project where given to us]. A short review of them would be:
+
+.High level requirements
```

Comment:
Form
Short textual description, probably in tabular use-case format.

Textual description, not table format